### PR TITLE
[WIP] Test build to check PR process

### PR DESCRIPTION
**Pull request by dmurphy18**
### What does this PR do?
Check PR build process

### What issues does this PR fix or reference?
Checking how PR build process works - test only

### CODE REVIEW

**Review by twangboy — APPROVED** [review]

**Comment by blu-base** [comment]
i believe, you just missed a single spot to bump 3005.1 to 3005.5, the respective suite name

https://github.com/dmurphy18/salt-bootstrap/blob/06c102b747c6a97f7186dbdf21d65597c4d5b1ae/kitchen.yml#L304

File: kitchen.windows.yml [hunk 1]
```diff
@@ -27,10 +27,10 @@ suites:
       salt_call_command: c:\Program Files\Salt Project\Salt\salt-call.bat
   - name: stable-3005
     provisioner:
-      salt_version: 3005.1-1
+      salt_version: 3005.5-1
   - name: stable-3005-1
```
Comment (blu-base) [inline kitchen.windows.yml:31]: this likely need's to be changed to `stable-3005-5`, too

File: kitchen.yml [hunk 2]
```diff
@@ -303,7 +303,7 @@ suites:
       - openbsd-6
   - name: stable-3005-1
```
Comment (blu-base) [inline kitchen.yml:304]: the valie need's to be changed to `stable-3005-5`

**Comment by dmurphy18** [comment]
@blu-base WIP Work In Progress :D , all of the 3005 support going away, 5 working days and it is EOL :partying_face: .
Kitchen-salt is going too, moving it to GitHub Actions, so still a lot of major changes to go
This is just a test to see what is failing, only changed the date.  Real PR is https://github.com/saltstack/salt-bootstrap/pull/1987

**Comment by blu-base** [comment]
Ok, nevermind. I was just trying to dive in a little bit, to see what I can help with in this project

**Comment by dmurphy18** [comment]
@blu-base Quick comment back, updated mine with real PR https://github.com/saltstack/salt-bootstrap/pull/1987

**Comment by dmurphy18** [comment]
Closing this since was a test of current build PR process, which has issues, see https://github.com/saltstack/salt-bootstrap/pull/1987 for actual PR doing the work

**Pull request closed without merging**